src/Lean/Server/FileWorker/WidgetRequests.lean

```diff
@@ -102,4 +102,12 @@ builtin_initialize
     (Array InteractiveDiagnostic)
     getInteractiveDiagnostics
 
+builtin_initialize
+  registerBuiltinRpcProcedure
+    `Lean.Widget.getGoToLocation
+    (FileWorker.GoToKind × WithRpcRef InfoWithCtx)
```

Comment:
```suggestion
    (FileWorker.GoToKind × WithRpcRef InfoWithCtx)
```
We should make this a structure so that we can add extra arguments in the future.